pkg/sif/descriptor.go

```diff
@@ -295,6 +300,24 @@ func (d Descriptor) SBOMMetadata() (SBOMFormat, error) {
 	return s.Format, nil
 }
 
+// OCIBlobMediaType returns the media for a OCI blob object.
+func (d Descriptor) OCIBlobMediaType() (string, error) {
```

Comment:
The name of this should correspond to `OptOCIBlobMetadata` so it's clear that they're related.

Also, it would be appropriate to use a named return here to make it clear to the caller what that `string` we're returning is.

Something like:

```suggestion
// OCIBlobMetadata gets metadata for a OCI blob data object.
//
//nolint:nonamedreturns // Named returns effective as documentation.
func (d Descriptor) OCIBlobMetadata() (mediaType string, err error) {
```